docs/madr/decisions/046-meshservice-multizone.md

```diff
@@ -0,0 +1,383 @@
+# MeshService multizone UX
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/9706
+
+## Context and Problem Statement
+
+With the introduction of `MeshService`, we need to decide how does it work with multizone deployments.
+
+There are two main use cases for multicluster connectivity:
+
+**Access service from another cluster**
+Service A is deployed only in zone X and the service in zone Y wants to consume it.
+
+**Replicate service over multiple clusters**
+Service A is deployed across multiple clusters for high availability.
+This strategy is used with "lift and shift" - migrating a workload from on prem to kubernetes.
+
+The current state is that a service is represented by `kuma.io/service` tag and as long as the tag is the same between zones, we consider this the same service.
+It means that a Kube service with the same name and namespace in two clusters is considered the same. 
+By default, if you don't have `MeshLoadbalancingStrategy` in the cluster, we load balance between local and remote zones. 
+
+This has a security disadvantage that if you take over the zone, you can advertise any service.
+
+## Considered Options
+
+* Access service from another cluster use case
+  * MeshService synced to other zones
+  * Service export / import
+    * Export field
+    * MeshTrafficPermission
+* Replicated services use case
+  * MeshService with dataplaneTags selector applied on global CP
+  * MeshMultiZoneService
+  * MeshService with meshServiceSelector selector applied on global CP
+
+## Decision Outcome
+
+Chosen options:
+* Access service from another cluster use case
+  * MeshService synced to other zones, because it's the only way
+  * Export field, because it's more explicit and more managable
+* Replicated services use case
+  * MeshMultiZoneService, with potentially diverging to "MeshService with meshServiceSelector selector applied on global CP" based on the future policy matching MADR.
+
+## Pros and Cons of the Options
+
+### Access service from another cluster use case
+
+#### MeshService synced to other zones
+
+MeshService is an object created on the zone that matches a proxies in this specific zone.
+This object is the synced to the global control plane.
+
+To solve the first use case we can then sync MeshService to other zones.
+Let's say we have the following MeshService in zone `east`
+```yaml
+kind: MeshService
+metadata:
+  name: redis
+  namespace: redis-system
+labels:
+  kuma.io/display-name: redis
+  k8s.kuma.io/namespace: redis-system
+  kuma.io/zone: east
+spec:
+  selector:
+    dataplaneTags:
```

Comment:
Should this be `spec.selector.dataplanes.tags` ? Maybe more future proof

Reply:
`dataplanes` or `dataplane`?